chainerio/filesystems/posix.py

```diff
@@ -43,7 +51,18 @@ def _recursive_list(self, prefix_end_index: int, path: str):
                                                 file.path)
 
     def stat(self, path):
-        return os.stat(path)
+        s = os.stat(path)
+        stat = PosixFileStat()
+        stat.mode = s.st_mode & 0x777
```

Comment:
Are you missing some fields here?
`filename`, `permission` (same as `mode`), `size`

Reply:
fixed in the latest commit! (2nd)